### PR TITLE
Fea/hampton crime

### PR DESCRIPTION
**Pull request by blackhatbrigade**

### CODE REVIEW

**Comment by blackhatbrigade** [comment]
Ready for a review @ezzy1337 @asedarski @krishnaramya 

This can be merged in, but we have one unit test failing and need some coverage on the new classes

File: src/HRQLS/Controllers/Crime/Hampton.php [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+        $response = new HerculesResponse();
+        $response->setEndpoint('/crime/Hampton');
+
+        $esResult = $app['elasticsearch']->search($this->indices, [], []);
+
+        $response = $this->parseResults($esResult, $response);
+
+
+        return $app->json($response->to_json(), $response->getStatusCode());
+        //return $app->json($response->to_json(), $response->getStatusCode());
```
Comment (asedarski) [inline src/HRQLS/Controllers/Crime/Hampton.php]: You should remove this :wink:

**Comment by asedarski** [comment]
:+1:

**Comment by blackhatbrigade** [comment]
Fixed the inconsistencies in the response object for the crime hampton endpoint

**Pull request merged**